### PR TITLE
Automated Goldpinger version update 3.7.0-5.5.0

### PR DESCRIPTION
**Pull request by replicated-ci-kurl**
Automated changes by the [cron-goldpinger-update](https://github.com/replicatedhq/kURL/blob/main/.github/workflows/update-goldpinger.yaml) GitHub action

```release-note
Adds [Goldpinger add-on](https://kurl.sh/docs/add-ons/goldpinger) version 3.7.0-5.5.0.
```

### CODE REVIEW

File: addons/goldpinger/3.6.1-5.4.3/install.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    local dst="$DIR/kustomize/goldpinger"
+
+    cp "$src/kustomization.yaml" "$dst/"
+    cp "$src/goldpinger.yaml" "$dst/"
+    cp "$src/servicemonitor.yaml" "$dst/"
+
+    if [ -n "${PROMETHEUS_VERSION}" ]; then
+        insert_resources "$dst/kustomization.yaml" servicemonitor.yaml
+    fi
+
```
Comment (rrpolanco) [inline addons/goldpinger/3.6.1-5.4.3/install.sh:13]: This monitor should be packaged with the prometheus add-on

Comment (emosbaugh) [reply addons/goldpinger/3.6.1-5.4.3/install.sh:13]: Perhaps we should add an addon_post_init hook to work around this dependency on the order the add-ons are applied?

**Comment by rrpolanco** [comment]
Diff: https://github.com/okgolove/helm-charts/compare/goldpinger-5.4.2...goldpinger-5.4.3

`5.4.3` allows a securityContext to be added to the daemonset.

**Review by rrpolanco — APPROVED** [review]

**Comment by laverya** [comment]
>Diff: https://github.com/okgolove/helm-charts/compare/goldpinger-5.4.2...goldpinger-5.4.3

>5.4.3 allows a securityContext to be added to the daemonset.

So skip this one, because we're not enabling security contexts and so it's exactly the same as the previous version?

**Pull request merged**